### ISSUE TITLE
Fitting gui() is not saving the fit values.

### ISSUE THREAD

**Issue by erh3cq** [hyperspy/hyperspy#1882]
I have been hand fitting some curves using the GUI(). If I switch to a new index, then go back to the index I just fit, the fit does not appear to have saved.

A few other comments have been made...
- @MurilooMoreira
In my case, when I change the parameters using m.gui() I can plot the new fitting and see the differences between the original fitting and the new one. And if I use the function m.print_current_values() I can see for sure the new sigma, center and A adjusted manually. But, when I use the function m.get_lines_intensity to obtain the counts I obtain the values from the original fitting. 
- @thomasaarholt
 I believe it is because the gui method does not call component.store_current_parameters_in_map() after changing the values

So my guess is although the interactive values are temporary (or local) and as Thomas says are not stored.

**Comment by thomasaarholt** [comment]
If `component.store_current_parameters_in_map()` is a bit slow to call whilst changing the values interactively, it could be possible to have two buttons, "Save values to model" and "Apply current values to all pixels" or similar. Not that the second one is necessary, but could be nice.

**Comment by erh3cq** [comment]
Yeah, that is what I was thinking. Having it try to update while moving the slider would be inefficient. I started looking throught the hyperspy_ipwidgets to add this, but could not figure it out.
Running the store comand does store the correct values.
